### DISCUSSION TITLE
MacOS SQLiteStudio Version wont install after download.

### DISCUSSION THREAD

**Discussion by AmandaOjay** [pawelsalawa/sqlitestudio#4675] [Q&A]
Downloaded the Mac OS version. But it won't let me install it. it says

"SQLiteStudio" cannot be opened because the developer cannot be verified. macOS cannot verify that this app is free from malware.The next prompt is to eject

**Comment by pawelsalawa** [comment]
Right-click on it and pick "Open". For the first time it will show the warning you have seen and just close the warning. Then right-click -> "Open" once again and it will show you the warning again, but this time you will have the button to open the app, despite the warning.

**Reply by AmandaOjay** [reply]
This didn't work for me. Is there anything else I can do?

**Reply by AmandaOjay** [reply]
Found a solution. Thanks

**Reply by avivash** [reply]
> Found a solution. Thanks

What was your solution?

**Reply by AmandaOjay** [reply]
I had to approve the installation from my system settings. Either under privacy or general settings

**Reply by avivash** [reply]
Thanks! I ended up doing the same thing too :)
